src/MCPClient/lib/clientScripts/pid_component_helpers.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+
+"""PID binding and declaration component helpers.
+
+Functions that straddle the capability of both sets of microservice jobs.
+"""
+import bindpid
+from main.models import Identifier
+
+
+class AssignPIDsException(Exception):
+    """Exception to raise if a validation or assignment error occurs."""
+
+
+def _add_custom_pid_to_mdl_identifiers(mdl, scheme, value):
```

Comment:
It seems like this would be most useful as a model method on the `SIP` `Directory`, and `File` models (so you can do `sip.add_custom_pid` or similar).

Reply:
I don't think we've been using model methods like this very often, but this is a nice suggestion. I like it.